frontend/src/routes/NotFound.tsx

```diff
@@ -50,6 +51,7 @@ export const NotFound: React.FC<Props> = ({ kind }) => {
     useNoindexTag();
 
     return <>
+        <Breadcrumbs path={[]} tail={<i>{title}</i>} />
```

Comment:
I'm really not sure if this page needs it? In the short text there is even a link to the homepage. That said, maybe lets just keep it for consistency reasons?

Reply:
Yes, it's not needed, but I also think keeping this is fine.